modules/components/pages/outputs/gcp_bigquery.adoc

```diff
@@ -225,8 +243,11 @@ The maximum number of bad records that will be ignored when reading data.
 
 === `auto_detect`
 
-Indicates if we should automatically infer the options and schema for CSV and JSON sources. If the table doesn't exist and this field is set to `false` the output may not be able to insert data and will throw insertion error. Be careful using this field since it delegates to the GCP BigQuery service the schema detection and values like `"no"` may be treated as booleans for the CSV format.
+Whether this component automatically infers the options and schema for `CSV` and `NEWLINE_DELIMITED_JSON` sources. 
+
+If this value is set to `false` and the destination table doesn't exist, the output may throw an insertion error as it is unable to insert data. 
```

Comment:
Why 'may'? What other conditions determine whether an error is thrown?

Reply:
Agreed. This error will occur. Updated to reflect this.